app/views/reports/form_partials/_check_box_tag_form_partial.html.haml

```diff
@@ -20,33 +20,31 @@
 - multiple        = options[:multiple]
 - grouping        = options[:grouping]
 - selected        = options[:selected] || []
-- default_classes = options[:required] ? "required" : ""
+- required_class = options[:required] ? "required" : ""
 - name            = options[:for] ? "report[#{options[:for]}]" : "report[#{field.name.underscore}_id]"
 - name           += "[]" if multiple
 - show_epic       = options[:field_label] == "Include Epic Interface Columns"
 
-= label_tag nil, field_label, class: ['col-sm-3 control-label', default_classes]
+= label_tag nil, field_label, class: ['mr-2', required_class]
 - if grouping && multiple # we have a defined grouping
-  .col-sm-9
-    - grouping.each do |k,v|
-      .row
-        %fieldset
-          %legend
-            = k
-          - v.each do |sv|
-            = label_tag sv, '', class: ['checkbox-inline', default_classes] do
-              = check_box_tag name, sv, selected.include?(sv), class: default_classes, type: "checkbox"
-              = multiple[sv]
+  - grouping.each do |k,v|
+    .row
+      %fieldset
+        %legend
+          = k
+        - v.each do |sv|
+          = label_tag sv, '', class: ['checkbox-inline', required_class] do
```

Comment:
The HTML `for` attribute isn't being set correctly so you can't click on the label to check/uncheck checkboxes.